model/src/main/kotlin/ScanSummary.kt

```diff
@@ -56,7 +57,13 @@ data class ScanSummary(
         /**
          * The list of errors that occurred during the scan.
          */
-        val errors: List<Error>
+        val errors: List<Error>,
+
+        /**
+         * A map that holds arbitrary data. Can be used by third-party tools to add custom data to the model.
+         */
+        @JsonInclude(JsonInclude.Include.NON_EMPTY)
+        val data: Map<String, Any> = emptyMap()
```

Comment:
Is "Mappers.kt " also considered  part of the public API? e.g. in order to use any type of objects a third party needs to set serializer via Mappers.kt?